docs/ff-concepts/animations/widget_animations.md

```diff
@@ -0,0 +1,232 @@
+---
+slug: widget-animations
+title: Widget Animations
+sidebar_position: 1
+description: Learn how to add widget animations in FlutterFlow.
+---
+import fadeGif from '@site/static/img/animations/animation_gifs/fade.gif';
+import scaleGif from '@site/static/img/animations/animation_gifs/scale.gif';
+import slideGif from '@site/static/img/animations/animation_gifs/slide.gif';
+import rotateGif from '@site/static/img/animations/animation_gifs/rotate.gif';
+import shakeGif from '@site/static/img/animations/animation_gifs/shake.gif';
+import blurGif from '@site/static/img/animations/animation_gifs/blur.gif';
+import saturateGif from '@site/static/img/animations/animation_gifs/saturate.gif';
+import tiltGif from '@site/static/img/animations/animation_gifs/tilt.gif';
+import flipGif from '@site/static/img/animations/animation_gifs/flip.gif';
+import shimmerGif from '@site/static/img/animations/animation_gifs/shimmer.gif';
+import tintGif from '@site/static/img/animations/animation_gifs/tint.gif';
+import animationCurves from '@site/static/img/animations/animation_gifs/animation_curves.gif';
+
+
+import animationsOverview from '@site/static/img/animations/animations_overview.png';
+import shimmerAnimationFinal from '@site/static/img/animations/animation_gifs/shimmerAnimationFinal.gif';
+import fadeEaseIn from '@site/static/img/animations/animation_gifs/fade-easein.gif';
+import fadeEaseInOut from '@site/static/img/animations/animation_gifs/fade-easeinout.gif';
+import fadeEaseOut from '@site//static/img/animations/animation_gifs/fade-easeout.gif';
+
+Widget animations allow you to add animation effects at the widget level.
+To add an animation to a widget, you'll need to go to the property panel for the widget and select the animations tab.
+
+<img src={animationsOverview}  alt="Animation tab within the property panel is used to add widget animations. Animation trigger specifies when the animation will be run. Widget animations can be triggered when the page (or widget in the case of a delayed load) loaded, or within an action. Animation effect specifies the type of animation to use.Animation curve specifies the function used to interpolate values over time. It controls the speed and style of the animation. Animation properties specifies the properties that can be configured for all widget animations, like whether or not it should loop. Effect-specific properties are properties to specific to the animation effect selected. For example, Fade animations allow you to control the initial opacity and the final opacity. You can customize these values to get the look and feel you desire." />
```

Comment:
In the image: "Effect-specific properties are properties to specific to the animation effect selected" should be grammatically be "Effect-specific properties are properties that are specific to the animation effect selected"

Reply:
Done!